log/syslog/syslog_writer.go

```diff
@@ -0,0 +1,23 @@
+package syslog
+
+import (
+	gosyslog "log/syslog"
+)
+
+// SyslogWriter is an interface wrapping stdlib syslog Writer
+type SyslogWriter interface {
+	Write([]byte) (int, error)
+	Close() error
+	Emerg(string) error
+	Alert(string) error
+	Crit(string) error
+	Err(string) error
+	Warning(string) error
+	Notice(string) error
+	Info(string) error
+	Debug(string) error
+}
+
```

Comment:
I would prefer all of the code in this file be combined with `syslog_logger.go`  and the combined file be renamed to `syslog.go`. I would put the `type SyslogWriter` declaration directly above the `func NewSyslogLogger` declaration.